src/main/java/org/jabref/gui/BasePanel.java

```diff
@@ -764,14 +769,57 @@ private void cut() {
      *            "deleted". If true the action will be localized as "cut"
      */
     private void delete(boolean cut) {
-        List<BibEntry> entries = mainTable.getSelectedEntries();
+    	List<FieldChange> changesRemove = new ArrayList<>();
+    	
+    	List<BibEntry> entries = mainTable.getSelectedEntries();
         if (entries.isEmpty()) {
             return;
         }
-        if (!cut && !showDeleteConfirmationDialog(entries.size())) {
-            return;
+    	
+        //create list of all groups of selected entries
+        Set<String> allEnriesGroups = new HashSet<String>(); 
+
+        for (BibEntry entrie : entries) {
+        	if (entrie.getField("groups").isPresent()) {
+        		System.out.println(entrie.getField("groups"));
```

Comment:
I guess this is for you to help in debugging, but in the final PR, there shouldn't be any `System.out` or commented-out code left. I'll not add a comment at every place in this PR, but I am sure you get the point.